include/CLI/Config.hpp

```diff
@@ -14,7 +14,7 @@
 #include <string>
 #include <utility>
 #include <vector>
-// [CLI11:public_includes:set]
+// [CLI11:public_includes:end]
```

Comment:
Wait, why did this work before, actually? It should search to the first "end", which I would have thought would have created a broken single header include. Testing locally, it does look like it just causes `#include <cctype>` to be missing; I guess this is included through something else so that's why it still works.

Reply:
After a brief look at the single header script, it seems that the multiline regex for needs both "set" and "end" comments. So it just didn't match and by lucky coincidence the necessary headers were included elsewhere.